packages/amino/src/pubkeys.ts

```diff
@@ -67,5 +67,5 @@ export interface MultisigThresholdPubkey extends Pubkey {
 }
 
 export function isMultisigThresholdPubkey(pubkey: Pubkey): pubkey is MultisigThresholdPubkey {
-  return (pubkey as MultisigThresholdPubkey).type === "tendermint/PubKeyMultisigThreshold";
+  return pubkey.type === "tendermint/PubKeyMultisigThreshold";
```

Comment:
Why did you remove those casts? They ensure we don't have a typo in the check. I know this is strange, but casting to the check type is the documented way of docing such checks.

Reply:
Oh! I just thought they were unnecessary. Makes sense though.

Reply:
If I was a compiler, I would optimize that out at compile time as always true. But TypeScript does not do that and does not seem to have plans to change that.